docs/components/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Divider as MuiDivider } from '@mui/material';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+export default {
+  title: 'Components/Divider',
+  component: MuiDivider,
```

Comment:
Do we still need to create a wrapper component for the divide? even we don't change nothing for the divider from \@mui/material, finally we need to provide ourselves divider component for the user.

Reply:
A wrapping component isn't necessary, library consumers can consume dividers directly from MUI. Additionally, some MUI components use the MUI Divider internally, so we want to ensure that the component and the theme overrides integrate with other MUI components.